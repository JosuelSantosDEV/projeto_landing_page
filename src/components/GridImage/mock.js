
const srcImg = "https://plus.unsplash.com/premium_photo-1673909405731-94b05ba69308?fm=jpg&q=60&w=3000&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8OXx8c291cmNlfGVufDB8fDB8fHww"
export default {
    background: false,
    title: "My grid",
    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit. 
    Accusantium commodi, vero quis
     atque architecto minus debitis nobis
     harum amet, illo, saepe enim velit eum a eveniet id aut totam quas.`,
     grid: [
        {
            altText: "Title 1",
            srcImg
        },
        {
            altText: "Title 2",
            srcImg
        },
        {
            altText: "Title 3",
            srcImg
        },
        {
            altText: "Title 4",
            srcImg
        },
        {
            altText: "Title 5",
            srcImg
        },
        {
            altText: "Title 6",
            srcImg
        },
     ]
}