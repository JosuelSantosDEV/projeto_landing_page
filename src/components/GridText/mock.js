export default {
    background: false,
    title: "My grid",
    description: "You’ve probably heard of Lorem Ipsum before – it’s the most-use",
    grid: [
        {
            title: "Test 1",
            description: `You’ve probably heard of Lorem Ipsum before – it’s the most-used dummy 
            text excerpt out there. People use it because it has a fairly normal distribution of 
            letters and words (making it look like normal English), but it’s also Latin, which 
            means your average reader won’t get distracted by trying to read it. It’s perfect for .`
        },
        {
            title: "Test 2",
            description: `You’ve probably heard of Lorem Ipsum before – it’s the most-used dummy 
            text excerpt out there. People use it because it has a fairly normal distribution of 
            letters and words (making it look like normal English), but it’s also Latin, which 
            means your average reader won’t get distracted by trying to read it. It’s perfect for .`
        },
        {
            title: "Test 3",
            description: `You’ve probably heard of Lorem Ipsum before – it’s the most-used dummy 
            text excerpt out there. People use it because it has a fairly normal distribution of 
            letters and words (making it look like normal English), but it’s also Latin, which 
            means your average reader won’t get distracted by trying to read it. It’s perfect for .`
        }
    ]
}