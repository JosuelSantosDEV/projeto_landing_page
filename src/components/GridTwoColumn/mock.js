export default {
    title:"Essa Ã© a GridTwoColumn",
    text: `Lorem ipsum dolor sit amet consectetur adipisicing elit. 
    Accusantium commodi, vero quis
     atque architecto minus debitis nobis
     harum amet, illo, saepe enim velit eum a eveniet id aut totam quas.`,
    srcImg: "assets/images/javascript.svg",
    background: false
}