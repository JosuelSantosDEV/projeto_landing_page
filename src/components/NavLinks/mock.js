export default [
    {
        children: "Home",
        link: "/home",
        newTab: true
    },
    {
        children: "About",
        link: "/about",
        newTab: true
    },
    {
        children: "Favorite",
        link: "/favorite",
        newTab: true
    }
]