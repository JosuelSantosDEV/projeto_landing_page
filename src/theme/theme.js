export const theme = {
    colors: {
        dark: "#000",
        white: "#fff",
        gray: "#c1c1c1"
    },
    fonts: {

    }
}