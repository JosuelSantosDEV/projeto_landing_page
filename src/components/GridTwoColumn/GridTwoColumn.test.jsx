import { renderTheme } from "../../theme/render-theme";
import { screen } from "@testing-library/react";
import { GridTwoColumn } from ".";
import mock from "./mock";

describe("<GridTwoColumn />", () => {

    it("Should render grid with two columns grid", () => {
        const {container} = renderTheme(<GridTwoColumn {...mock} ></GridTwoColumn>);
        expect(container).toMatchSnapshot()
    }
    );

});