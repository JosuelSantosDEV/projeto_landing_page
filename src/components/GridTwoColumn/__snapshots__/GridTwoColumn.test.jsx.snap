// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridTwoColumn /> Should render grid with two columns grid 1`] = `
.c5 {
  color: #0a1128;
  font-size: 3.2rem;
  text-transform: uppercase;
}

.c2 {
  display: grid;
  grid-template-columns: 1fr 2fr;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 3.2rem;
}

.c2 .c4 {
  margin-bottom: 3.2rem;
}

.c0 {
  background: #fff;
  color: #0a1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c6 {
  font-size: 2.4rem;
}

@media (max-width:768px) {
  .c2 {
    grid-template-columns: 1fr;
    text-align: center;
  }
}

@media (max-width:768px) {
  .c3 {
    margin-bottom: 3.2rem;
  }
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <div>
            <h2
              class="c4 c5"
            >
              Essa é a GridTwoColumn
            </h2>
          </div>
          <p
            class="c6"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. 
    Accusantium commodi, vero quis
     atque architecto minus debitis nobis
     harum amet, illo, saepe enim velit eum a eveniet id aut totam quas.
          </p>
        </div>
        <div
          class=""
        >
          <img
            alt="Essa é a GridTwoColumn"
            class=""
            src="assets/images/javascript.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
